夏季实训pr

wegul opened the pull request:
https://aistudio.baidu.com/aistudio/projectdetail/2240858?shared=1

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/awesome-DeepLearning?pullRequest=654) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/awesome-DeepLearning?pullRequest=654) before we can accept your contribution.<br/>**8** out of **9** committers have signed the CLA.<br/><br/>:white_check_mark: tngt<br/>:white_check_mark: chajchaj<br/>:white_check_mark: lvjian0706<br/>:white_check_mark: w5688414<br/>:white_check_mark: XYZ916829<br/>:white_check_mark: an1018<br/>:white_check_mark: ZhangHandi<br/>:white_check_mark: youngzhou1999<br/>:x: milen<br/><hr/>**milen** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/awesome-DeepLearning?pullRequest=654) it.</sub>

tngt committed:
Merge branch 'PaddlePaddle:master' into awe

JayJayMing committed:
Create 8.1sequence.md

FeiMa-REC committed:
#d2l add chapter 11.3 11.7 11.8 11.9 12.7 and update paddle.py

JiehangXie committed:
#d2l update chapter8.2 8.3 13.10 13.12

更新markdown，删除输出内容

skywalk163 committed:
1

1

skywalk163 committed:
2

2

skywalk163 committed:
1

1

skywalk163 committed:
12.1章节

上传

JiehangXie committed:
#d2l add chap 9.5 and update d2l

skywalk163 committed:
2

2

skywalk163 committed:
12.2

JIANG-HS committed:
add 8.5 8.6

add 8.5 8.6

zhangjinCV committed:
Add files via upload

zhangjinCV committed:
Update ParamsCounter.md

JayJayMing committed:
Create gru.md

JayJayMing committed:
Delete gru.md

JayJayMing committed:
Create gru.md

JayJayMing committed:
Create lstm.md

9.2

JayJayMing committed:
Create deep-rnn.md

9.3

JayJayMing committed:
Create bi-rnn.md

9.4

skywalk163 committed:
12.3

JiehangXie committed:
#d2l add chap 9.8

JiehangXie committed:
#d2l add chap9.6 and update d2l

anliyuan committed:
10.2,10.3,10.5,10.6

Shelly111111 committed:
#d2l add 9.7 chapter

chajchaj committed:
Merge pull request #814 from liucongg/master

SimCSE论文讲解完善

JayJayMing committed:
Update 8.1sequence.md

anliyuan committed:
提交10.4 10.7，修改10.5中的bug

tngt committed:
Merge branch 'PaddlePaddle:master' into awe

tngt committed:
#d2l add FAQ

skywalk163 committed:
12.5

JIANG-HS committed:
add 14.5 & 14.6

add 14.5 & 14.6

chajchaj committed:
Merge pull request #834 from tngt/awe

add faq

anliyuan committed:
multihead-attention中的bug修改

skywalk163 committed:
12.6

JayJayMing committed:
Update gru.md

JayJayMing committed:
Update lstm.md

JayJayMing committed:
Update deep-rnn.md

JayJayMing committed:
Update bi-rnn.md

tensorfly-gpu committed:
#d2l add 11.6momentum.md

tensorfly-gpu committed:
#d2l add 11.5minibatch-sgd.md

tensorfly-gpu committed:
#d2l add 11.1 optimization-intro.md

tensorfly-gpu committed:
#d2l add 11.2convexity.md

tensorfly-gpu committed:
#d2l add 11.4sgd.md

Shelly111111 committed:
#d2l update 9.7 chapter

Shelly111111 committed:
#d2l update 9.7 chapter

Commit:
new delete

Junxiao-Ma committed:
Add files via upload

JIANG-HS committed:
add 14.4

add 14.4

Junxiao-Ma committed:
Add files via upload

FeiMa-REC committed:
updated section 11.3 11.7 11.8 11.9, use d2l.* instead paddle.*

FeiMa-REC committed:
updated the functions of section 11.3 in paddle.py

anliyuan committed:
10.7bug

JayJayMing committed:
Update gru.md

JayJayMing committed:
Update lstm.md

Commit:
add nlp_tasks

Commit:
mv nlp_tasks to Paddle_Industry_Practice_Sample_Library

Commit:
rename nlp_tasks to nlp_projects

zhangjinCV committed:
Update ParamsCounter.md

删除图片，写出公式。修改一个错误

chajchaj committed:
Merge pull request #840 from 1649759610/master

add 7 nlp projects

chajchaj committed:
Merge pull request #810 from zhangjin12138/master

CNN 模型所需的计算力（flops）和参数（parameters）数量计算（给出计算推导、例子）

an1018 committed:
add MultimodalVideoTag

Commit:
add sentiment_analysis

ZhaoJingjing713 committed:
Add Figure_Skating in Paddle_Industry_Practice_Sample_Library

chajchaj committed:
Merge pull request #842 from an1018/add_info

add MultimodalVideoTag

chajchaj committed:
Merge pull request #846 from ZhaoJingjing713/my_branch

Add Figure_Skating in Paddle_Industry_Practice_Sample_Library

Junxiao-Ma committed:
Add files via upload

JIANG-HS committed:
add 13.5.md

add 13.5.md

JiehangXie committed:
#d2l add chap 14.9

ZhangHandi committed:
add industry practice sample library

tngt committed:
Merge pull request #848 from ZhangHandi/footballaction

add industry practice sample library

tngt committed:
update README

tngt committed:
Merge pull request #849 from tngt/awe

update README

tngt committed:
update README

tngt committed:
Merge pull request #850 from tngt/awe

update README

ZhangHandi committed:
modify football action

tngt committed:
Merge pull request #851 from ZhangHandi/footballaction

modify football action

an1018 committed:
modify MultimodalVideoTag

an1018 committed:
modify MultimodalVideoTag

an1018 committed:
modify MultimodalVideoTag

chajchaj committed:
Merge pull request #831 from anliyuan/d2l

10.2,10.3,10.5,10.6

chajchaj committed:
Merge pull request #838 from tensorfly-gpu/master

#d2l add11.1,11.2,11.4,11.5,11.6

FeiMa-REC committed:
update 11.3 11.8 and optimize code format in paddle.py

Shelly111111 committed:
#d2l add 13.11 chapter

chajchaj committed:
Merge pull request #852 from an1018/add_info

modify MultimodalVideoTag

ZhangHandi committed:
modify football action file and readme

ZhangHandi committed:
update multimodelvideotag readme

ZhangHandi committed:
modify readme

ZhangHandi committed:
resolve conflict

chajchaj committed:
Merge pull request #843 from 1649759610/master

add sentiment_analysis

Shelly111111 committed:
#d2l add 13.13 chapter

chajchaj committed:
Merge pull request #854 from ZhangHandi/footballaction

modify football action file and readme

Shelly111111 committed:
#d2l add 13.14 chapter

JayJayMing committed:
Create 13.9 semantic-segmentation-and-dataset.md

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

Junxiao-Ma committed:
Add files via upload

XYZ-916 committed:
add abnormal action detection case

zbp-xxxp committed:
update 13.8 R-CNN

#d2l 13.8区域卷积神经网络(R-CNN)系列_副本
https://aistudio.baidu.com/aistudio/projectdetail/3499998?forkThirdPart=1

Junxiao-Ma committed:
Add files via upload

JiehangXie committed:
#d2l add chap 2.6

lvjian0706 committed:
add Multi-Class Vehicle Tracking

lvjian0706 committed:
add Multi-Class Vehicle Tracking

chajchaj committed:
Merge pull request #856 from XYZ-916/master

add abnormal action detection case

chajchaj committed:
Merge pull request #859 from lvjian0706/add_docs

add Multi-Class Vehicle Tracking

tngt committed:
update README

tngt committed:
update README

skywalk163 committed:
12.1 12.2 12.5 

12.1：
1.“通过混合式编程加速”：时间测试结果与结论不符，应该是静态图时间更短，可以确认下代码是否有问题。 
答：会再次确认。个人认为是因为小模型，并且静态图最后结果会回传到动态图导致的消耗所致，报issue咨询
12.2:
1.“通过后端异步处理”：文档是pytorch，可以看下能否调整成paddle的说明 。 
答：前期是讲的不调文档。这里已经做了相应修改
2.“障碍器与阻碍器”、“改进计算”：这部分没有写？
答：因为前期沟通的是修改torch部分，这部分torch没有写，所以才没写。这部分已经加上。
12.5：
1.“数据并行性”：文档中“在实践中请注意，当在k个GPU上训练时，需要扩大小批量的大小为k的倍数”在paddle中不适用
答：已经进行修改。
2.“训练”中epoch设置与原版不统一
答：已修改成跟原版一样。

skywalk163 committed:
Delete 12_1.ipynb

修改成12.1

skywalk163 committed:
Delete 12_1.md

修改成12.1

chajchaj committed:
Merge pull request #860 from tngt/awe

update README

tngt committed:
Merge pull request #827 from YuBCS-Ma/YuBCS-Ma_pr

#d2l add chapter 11.3 11.7 11.8 11.9 12.7 and update paddle.py

skywalk163 committed:
12.6 

1.“网络初始化”：建议与原版保持一致，否则这一部分没有出现的必要
答：还是放入训练部分了。
2.原版使用的是d2l的评估代码，这里为什么直接调用的paddle？
答：这里既用了d2l的评估代码，也使用了飞桨自己了，是因为飞桨的多GPU简洁实现必须在命令行下实现，这样d2l的图形实现无法显示而失效，所以才添加了飞桨自己的。
3.训练时长没有计算和打印
答：这块已加上。

tngt committed:
update README

tngt committed:
Merge pull request #863 from tngt/awe

update README

Shelly111111 committed:
#d2l add 13.7 chapter

Shelly111111 committed:
update d2l/paddle.py

JiehangXie committed:
#d2l update chap 5.5

Zhiyuan-Fan committed:
15.3

Zhiyuan-Fan committed:
15.4

Zhiyuan-Fan committed:
15.6

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

Shelly111111 committed:
#d2l update 13.11 chapter fcn.md

Evezerest committed:
Add Electricity_Meter_Recognition

Add Electricity_Meter_Recognition

tngt committed:
Merge pull request #867 from Evezerest/master

Add Electricity_Meter_Recognition

Evezerest committed:
Merge remote-tracking branch 'PaddlePaddle/master'

Evezerest committed:
Change File Location

Change File Location

tngt committed:
Merge pull request #868 from Evezerest/master

Change File Location

tngt committed:
update README

tngt committed:
update README

tngt committed:
Merge pull request #869 from tngt/awe

update README

JiehangXie committed:
#d2l update chap 5.2

JIANG-HS committed:
删除device & 去除cpu()

tngt committed:
update readme

tngt committed:
Merge pull request #870 from tngt/awe

update readme

zbp-xxxp committed:
Update 13.8_RCNN.ipynb

zbp-xxxp committed:
Merge branch 'PaddlePaddle:master' into master

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

skywalk163 committed:
12.6

修改成spawn并行

JiehangXie committed:
#d2l update chap 13.10&13.12

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

Shelly111111 committed:
#d2l update 13.13 and 13.14 chapter

skywalk163 committed:
创建目录

skywalk163 committed:
提交14.8和14.10

bert哦！

skywalk163 committed:
Merge branch 'PaddlePaddle:master' into master

Junxiao-Ma committed:
Add files via upload

Junxiao-Ma committed:
Add files via upload

skywalk163 committed:
12.1 修改

Shelly111111 committed:
#d2l update 13.7 ./img->../img and 13.11 ./img->../img

tngt committed:
Merge pull request #830 from JIANG-HS/d2l

#d2l add 8.5 8.6 13.5 14.4 14.5 14.6

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

Shelly111111 committed:
#d2l delete del/paddle.py  conflict resolution

Shelly111111 committed:
#d2l add d2l/paddle.py conflict solution

tngt committed:
Merge pull request #820 from Shelly111111/d2l

#d2l add 8.4 chapter rnn.md

skywalk163 committed:
12.6

从spawn修改为launch模式

skywalk163 committed:
14.8 14.10 

删除冗余和注释行。
提升了速度。
精度提高还没找到思路，需要大家会诊一下了。

skywalk163 committed:
14.8

去掉空行

Junxiao-Ma committed:
Add files via upload

Zhiyuan-Fan committed:
修改初始化方式

skywalk163 committed:
修改step到50

JiehangXie committed:
#d2l update chap 14.9

Zhiyuan-Fan committed:
attention

Zhiyuan-Fan committed:
修改

Junxiao-Ma committed:
Add files via upload

Junxiao-Ma committed:
Add files via upload

Junxiao-Ma committed:
Add files via upload

skywalk163 committed:
12.3 

尽量跟torch代码一致。正式版和develop版暂时都无法跑通，需要版本支持to_tensor放置指定gpu处。

an1018 committed:
add tower_rec

tngt committed:
Merge pull request #871 from an1018/add_info

add tower_rec

tngt committed:
update README

tngt committed:
Merge pull request #872 from tngt/awe

update README

tngt committed:
update README

tngt committed:
Merge pull request #873 from tngt/awe

update README

an1018 committed:
modify readme

tngt committed:
Merge pull request #874 from an1018/add_info

modify readme

skywalk163 committed:
12.3 

计算用全局变量放置在相对应的gpu上。

JayJayMing committed:
Update 13.9 semantic-segmentation-and-dataset.md

skywalk163 committed:
12.5

使用全局变量让所有gpu都参与运算。

JiehangXie committed:
#d2l update chap 14.9

skywalk163 committed:
12.5

skywalk163 committed:
12.3 

修改了这里     paddle.set_device(f"gpu:{index}")

tensorfly-gpu committed:
#d2l add 13.6

tngt committed:
Merge pull request #882 from tensorfly-gpu/master

#d2l add 13.6

tensorfly-gpu committed:
#d2l update 11.2, 11.5, 11.6

tensorfly-gpu committed:
#d2l update paddle.py

所有章节已开发完毕，接下来的修改需要大家一起来把关了

tngt committed:
Merge pull request #883 from tensorfly-gpu/master

#d2l update 11.2, 11.5, 11.6, update paddle.py

tngt committed:
add smart_drive

tngt committed:
Merge pull request #884 from tngt/awe

add smart_drive

skywalk163 committed:
14.10

加上单机多卡程序。

tngt committed:
Merge pull request #839 from Junxiao-Ma/master

#d2l  add  11_10 11_11 13_1 13_2 14_7

tngt committed:
Merge pull request #828 from skywalk163/master

#d2l  12.1 12.2 12.3 12.5 12.6 14.8 14.10

Zhiyuan-Fan committed:
Merge branch 'PaddlePaddle:master' into master

JiehangXie committed:
#d2l delete paddle.py

tngt committed:
update README

tngt committed:
Merge pull request #885 from tngt/awe

update README

JiehangXie committed:
#d2l update paddle.py

JiehangXie committed:
Merge branch 'master' into d2l

tngt committed:
Merge pull request #817 from JiehangXie/d2l

#d2l add chapter 8.2 and fix temporary path in d2l

Zhiyuan-Fan committed:
Merge branch 'PaddlePaddle:master' into master

hiweiwu committed:
#d2l update no conflict version

tngt committed:
Merge pull request #886 from nsCosine/d2l

#d2l update 13.3,13.4,14.1,14.2,14.3 no conflict version

tngt committed:
Merge pull request #826 from jiejieyun/d2l

#d2l

tngt committed:
Merge pull request #865 from Elvisambition/master

15.3

JIANG-HS committed:
8.5 & 8.6 删除不必要的device

JayJayMing committed:
Update lstm.md

JIANG-HS committed:
处理冲突

tngt committed:
Merge branch 'master' into d2l

tngt committed:
Merge pull request #887 from JIANG-HS/d2l

8.5 & 8.6 & paddle.py 删除不必要的device

JayJayMing committed:
Update gru.md

JIANG-HS committed:
删除不必要代码

JIANG-HS committed:
Merge branch 'd2l' of https://github.com/JIANG-HS/awesome-DeepLearning into d2l

tngt committed:
Merge pull request #889 from jiejieyun/patch-4

#d2l Update gru.md

tngt committed:
Merge pull request #888 from jiejieyun/patch-3

#d2l  Update lstm.md

JIANG-HS committed:
删除paddle.to_tensor

tngt committed:
Merge pull request #890 from JIANG-HS/d2l

删除paddle.to_tensor

JIANG-HS committed:
8.5忘删了一个device

tngt committed:
Merge pull request #891 from JIANG-HS/d2l

8.5忘删了一个device

tngt committed:
delete extra 15chap

tngt committed:
Merge pull request #892 from tngt/awe

delete extra 15chap

tngt committed:
Handle 8-15 chapter directory names

tngt committed:
Merge pull request #893 from tngt/awe

Unified 8-15 chapter directory names

tngt committed:
Merge pull request #857 from zbp-xxxp/master

update 13.8 R-CNN

tngt committed:
add case of Fire_and_Smoke_Detection

tngt committed:
Merge pull request #898 from tngt/awe

add case of Fire_and_Smoke_Detection

tngt committed:
update README

tngt committed:
Merge pull request #899 from tngt/awe

update README

tngt committed:
Update README.md

ZhangHandi committed:
add 3d detection case

chajchaj committed:
Merge pull request #901 from ZhangHandi/my_branch

add 3d detection case

icm-ai committed:
Update GPT.md

icm-ai committed:
Update transformer.md

修复错字

icm-ai committed:
Update GPT.md

icm-ai committed:
Update bert.md

icm-ai committed:
Update erine.md

icm-ai committed:
Update Performer.md

kxxt committed:
Fix math rendering issue in adadelta.md

tngt committed:
Update README.md

hua-zi committed:
【文档修复】Update 5.2.md

5.2.2.1. 内置初始化

使用 set_value() 初始化参数

tngt committed:
Update README.md

SamZhangQingChuan committed:
Fix typo in pretrain_model_description.md

jzhang533 committed:
Merge pull request #924 from SamZhangQingChuan/patch-1

Fix typo in pretrain_model_description.md

jzhang533 committed:
Merge pull request #922 from hua-zi/patch-1

【文档修复】Update 5.2.md

jzhang533 committed:
Merge pull request #920 from kxxt/patch-1

Fix math rendering issue in adadelta.md

jzhang533 committed:
Merge pull request #919 from icm-ai/patch-6

Update Performer.md

jzhang533 committed:
Merge pull request #918 from icm-ai/patch-5

Update erine.md

jzhang533 committed:
Merge pull request #917 from icm-ai/patch-4

Update bert.md

jzhang533 committed:
Merge pull request #916 from icm-ai/patch-3

Update GPT.md

jzhang533 committed:
Merge pull request #915 from icm-ai/patch-2

Update transformer.md

jzhang533 committed:
Merge pull request #914 from icm-ai/patch-1

Update GPT.md
